sys/sys/thr.h

```diff
@@ -55,6 +55,7 @@ struct thr_param {
     long	*parent_tid;		/* parent accesses the new TID here. */
     int		flags;			/* thread flags. */
     struct rtprio	*rtp;		/* Real-time scheduling priority */
+    /* XXXAR: should ddc always be here? It is only used by CHERIABI. */
```

Comment:
Should I just write into spare[0] in libthr instead and add a comment that this is ddc in CHERIABI?
Or is the ifdef the better solution?